buildSrc/src/main/kotlin/multiplatform-module.gradle.kts

```diff
@@ -64,8 +64,18 @@ tasks {
     getByName<KotlinJvmTest>("jvmTest") {
         useJUnitPlatform()
     }
+    withType<KotlinNativeSimulatorTest>() {
+        enabled = false
+    }
```

Comment:
why this?

Reply:
The cloud based MacOS runners don't have the simulator runtimes installed

Reply:
i see